include/types.h

```diff
@@ -25,6 +25,10 @@ struct mako_directional {
 };
 
 bool parse_directional(const char *string, struct mako_directional *out);
+
+// List of specifier characters that can appear in a format string.
+static const char VALID_FORMAT_SPECIFIERS[] = "%asbht";
```

Comment:
I'd like not to introduce `static` definitions in headers. Instead we can declare this constant as `extern` here and define it in a `.c` file.